docs/content/src/learn/tutorials/saas-integrations/gmail/working-with-googlespreadsheet-service/README.md

```diff
@@ -0,0 +1,194 @@
+
+A Ballerina project is created for the integration use case explained above. Please follow the steps given 
+below to create the project and modules. You can learn about the Ballerina project and modules in this 
+[guide](https://ei.docs.wso2.com/en/latest/ballerina-integrator/develop/using-modules/#creating-a-project).
+
+#### 1. Create a new project.
+
+```bash
+$ ballerina new working-with-googlespreadsheet-service
+```
+
+#### 2. Create a module.
+
+```bash
+$ ballerina add googlespreadsheet_service
+```
+
+The project structure is created as indicated below.
+
+```
+working-with-googlespreadsheet-service
+    ├── Ballerina.toml
+    └── src
+        └── googlespreadsheet_service
+            ├── Module.md
+            ├── main.bal
+            ├── resources
+            └── tests
+                └── resources
+```
+
+#### 3. Set up credentials for accessing Amazon S3
+
+- Visit [Credentials](https://console.developers.google.com/apis/credentials) to get client ID, client secret and other required credentials from Google.
+
+- You can also follow [Get Credentials for Google Spreadsheet](https://docs.wso2.com/display/IntegrationCloud/Get+Credentials+for+Google+Spreadsheet) to get the required credentials for this application.
+
+#### 4. Add project configurations file
+
+Add the project configuration file by creating a `ballerina.conf` file under the root path of the project structure. 
+This file should have following configurations. Add the obtained Amazon S3 configurations to the file.
+
+```
+ACCESS_TOKEN="<Acces_token>"
+CLIENT_ID="<Client_id">
+CLIENT_SECRET="<Client_secret>"
+REFRESH_TOKEN="<Refresh_token>"
+REFRESH_URL = "<Refresh_URL>"
+LISTENER_PORT = Listener_port>
+BASE_PATH = "/spreadsheets"
+```
+
+#### 5. Write the integration
+Open the project with VS Code. The integration implementation is written in the `src/googlespreadsheet_service/main.bal` file.
+
+<!-- INCLUDE_CODE: src/googlespreadsheet_service/main.bal -->
+
+## Testing 
+
+First let’s build the module. Navigate to the project root directory and execute the following command.
+
+```bash
+$ ballerina build googlespreadsheet_service
+```
+
+This creates the executables. Now run the `googlespreadsheet_servicet.jar` file created in the above step.
+
+```bash
+$ java -jar target/bin/googlespreadsheet_service.jar
+```
+
+You will see the following service log after successfully invoking the service.
+
+```log
+[ballerina/http] started HTTP/WS listener 0.0.0.0:9090
+```
+
+### 1. Testing create a new spreadsheet
+ ```bash
+ curl -v -X POST http://localhost:9090/spreadsheets/<SPREADSHEET_NAME>
+```
+e.g.   
+```bash 
+curl -v -X POST http://localhost:9090/spreadsheets/firstSpreadsheet
+```
+### 2. Testing adding a new worksheet
+  ```bash
+        curl -v -X POST http://localhost:9090/spreadsheets/<SPREADSHEET_ID>/<WORKSHEET_NAME>
+
+        e.g: curl -v -X POST http://localhost:9090/spreadsheets/1AoOHLyn3Ds6do6UMq8t_pv20RrRwNV4aoqQVI_Z5xKY/firstWorksheet
```

Comment:
Let's put this out of the bash command block. Also use "e.g.", the dot at the end is necessary since this is a latin derivation.

Reply:
Fixed